[2024-08-12T18:04:46.536+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T18:04:46.551+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T18:04:46.551+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T18:04:46.660+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2024-03-01 00:00:00+00:00
[2024-08-12T18:04:46.714+0000] {standard_task_runner.py:57} INFO - Started process 7834 to run task
[2024-08-12T18:04:46.764+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_table', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmpnfvd2e86']
[2024-08-12T18:04:46.871+0000] {standard_task_runner.py:85} INFO - Job 202: Subtask create_table
[2024-08-12T18:04:47.605+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T18:04:48.168+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-08-12T18:04:48.173+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/monitor.py", line 44, in create_table
    conn = psycopg2.connect(**conn_params)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?

[2024-08-12T18:04:48.318+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=monthly_monitor, task_id=create_table, execution_date=20240301T000000, start_date=20240812T180446, end_date=20240812T180448
[2024-08-12T18:04:48.376+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 202 for task create_table (connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
; 7834)
[2024-08-12T18:04:48.496+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-08-12T18:04:48.955+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T18:13:33.347+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T18:13:33.402+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T18:13:33.422+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T18:13:33.508+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2024-03-01 00:00:00+00:00
[2024-08-12T18:13:33.535+0000] {standard_task_runner.py:57} INFO - Started process 9100 to run task
[2024-08-12T18:13:33.544+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_table', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmp3l40957x']
[2024-08-12T18:13:33.557+0000] {standard_task_runner.py:85} INFO - Job 254: Subtask create_table
[2024-08-12T18:13:34.063+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T18:13:34.566+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-08-12T18:13:34.599+0000] {logging_mixin.py:151} INFO - table monitor_metrics already exists
[2024-08-12T18:13:34.607+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-08-12T18:13:34.655+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_table, execution_date=20240301T000000, start_date=20240812T181333, end_date=20240812T181334
[2024-08-12T18:13:35.035+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T18:13:35.253+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T19:22:10.923+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T19:22:10.979+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T19:22:10.991+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T19:22:11.070+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2024-03-01 00:00:00+00:00
[2024-08-12T19:22:11.168+0000] {standard_task_runner.py:57} INFO - Started process 4352 to run task
[2024-08-12T19:22:11.183+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_table', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '494', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmp7_teowj_']
[2024-08-12T19:22:11.203+0000] {standard_task_runner.py:85} INFO - Job 494: Subtask create_table
[2024-08-12T19:22:11.390+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T19:22:12.005+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-08-12T19:22:12.155+0000] {logging_mixin.py:151} INFO - table monitor_metrics already exists
[2024-08-12T19:22:12.166+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-08-12T19:22:12.204+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_table, execution_date=20240301T000000, start_date=20240812T192210, end_date=20240812T192212
[2024-08-12T19:22:12.572+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T19:22:12.626+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T19:27:21.450+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T19:27:21.474+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T19:27:21.490+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T19:27:21.537+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2024-03-01 00:00:00+00:00
[2024-08-12T19:27:21.617+0000] {standard_task_runner.py:57} INFO - Started process 5368 to run task
[2024-08-12T19:27:21.633+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_table', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '545', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmpbg0k25qm']
[2024-08-12T19:27:21.673+0000] {standard_task_runner.py:85} INFO - Job 545: Subtask create_table
[2024-08-12T19:27:21.939+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T19:27:22.268+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-08-12T19:27:22.306+0000] {logging_mixin.py:151} INFO - table monitor_metrics already exists
[2024-08-12T19:27:22.307+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-08-12T19:27:22.376+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_table, execution_date=20240301T000000, start_date=20240812T192721, end_date=20240812T192722
[2024-08-12T19:27:22.434+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T19:27:22.462+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T19:28:43.666+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T19:28:43.684+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T19:28:43.687+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T19:28:43.707+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2024-03-01 00:00:00+00:00
[2024-08-12T19:28:43.718+0000] {standard_task_runner.py:57} INFO - Started process 5876 to run task
[2024-08-12T19:28:43.725+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_table', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '587', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmpq7zzkddh']
[2024-08-12T19:28:43.732+0000] {standard_task_runner.py:85} INFO - Job 587: Subtask create_table
[2024-08-12T19:28:43.812+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T19:28:43.930+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-08-12T19:28:43.939+0000] {logging_mixin.py:151} INFO - table monitor_metrics already exists
[2024-08-12T19:28:43.939+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-08-12T19:28:43.951+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_table, execution_date=20240301T000000, start_date=20240812T192843, end_date=20240812T192843
[2024-08-12T19:28:43.997+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T19:28:44.016+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T20:10:14.797+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T20:10:14.832+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-08-12T20:10:14.837+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T20:10:14.901+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2024-03-01 00:00:00+00:00
[2024-08-12T20:10:14.932+0000] {standard_task_runner.py:57} INFO - Started process 13210 to run task
[2024-08-12T20:10:14.937+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_table', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '918', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmp3xa8x8gm']
[2024-08-12T20:10:14.961+0000] {standard_task_runner.py:85} INFO - Job 918: Subtask create_table
[2024-08-12T20:10:15.185+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_table scheduled__2024-03-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T20:10:15.535+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-08-12T20:10:15.555+0000] {logging_mixin.py:151} INFO - table monitor_metrics already exists
[2024-08-12T20:10:15.557+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-08-12T20:10:15.571+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_table, execution_date=20240301T000000, start_date=20240812T201014, end_date=20240812T201015
[2024-08-12T20:10:15.664+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T20:10:15.700+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
